Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 gem "github-pages", group: :jekyll_plugins
 gem "jekyll-paginate"
+
+gem "webrick", "~> 1.7"
```

Comment:
adding this in the gemfile because it is very often required to install it anyway.